packages/components/src/stepper/index.js

```diff
@@ -79,6 +88,21 @@ Stepper.propTypes = {
             isComplete: PropTypes.bool,
 		} )
 	).isRequired,
+
+	/**
+	 * Direction of the stepper.
+	 */
+	direction: PropTypes.oneOf( [ 'horizontal', 'vertical' ] ),
+
+	/**
+	 * Optional mark the current step as in pending to show a spinner treatment.
```

Comment:
Minor typo:
```suggestion
	 * Optionally mark the current step as pending to show a spinner.
```

Reply:
Wow that original sentence. I blame Friday.